packages/internal/src/paths.ts

```diff
@@ -193,3 +193,15 @@ export const processPagesDir = (
   })
   return deps
 }
+
+export const toPosixPathOnWindows = (path: string) => {
```

Comment:
If anyone has a better name for this function I'm all ears 👂

Reply:
Make something like `ensurePosixPath`